code/modules/mob/dead/observer/verbs.dm

```diff
@@ -519,6 +519,35 @@
 	else
 		to_chat(src, "<span class='warning'>Unable to find any MoMMI Spawners ready to build a MoMMI in the universe. Please try again.</span>")
 
+/mob/dead/observer/verb/become_hobo()
+	set name = "Become Space Hobo"
+	set category = "Ghost"
+	if(!can_reenter_corpse)
+		to_chat(src, "<span class='warning'>You must have had presence on this plane to become this.</span>")
+		return
+	if(!config.respawn_as_hobo)
+		to_chat(src, "<span class='warning'>Respawning as Space Hobo is disabled.</span>")
+		return
+
+	var/timedifference = world.time - client.time_died_as_mouse
+	if(client.time_died_as_mouse && timedifference <= mouse_respawn_time * 600)
+		var/timedifference_text
+		timedifference_text = time2text(mouse_respawn_time * 600 - timedifference,"mm:ss")
+		to_chat(src, "<span class='warning'>You may only spawn again as this role more than [mouse_respawn_time] minutes after your death. You have [timedifference_text] left.</span>")
+		return
+
+	if(!hobostart.len || hobostart.len < 9)
+		generate_hoboshack()
+	//find a viable mouse candidate
+	var/mob/living/carbon/human/hobo = Humanize(pick(200;"Human",50;"Vox",50;"Insectoid",25;"Diona",25;"Grey",1;"Tajaran",10;"Unathi"))
+	hobo.forceMove(pick(hobostart))
```

Comment:
this is even odder because it doesn't put the new hobo for sure in the new shack -- it could spawn a new shack and then put them in the first shack, it only guarantees there is 1 shack per each hobo (up to 8)

Reply:
Intended, but I guess not ideal to spawn empty shacks

Reply:
another thing I notice on here: 1% catbeast is funny but also means they will spawn with unwearable magboots; consider giving them the special catbeast sandals (which are not special but at least they won't be shoeless)

Reply:
Done